### PR TITLE
migrate: components_polymer3/vz_sorting

### PR DESCRIPTION
**Pull request by psybuzz**

### CODE REVIEW

File: tensorboard/components_polymer3/vz_sorting/BUILD [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
     name = "vz_sorting",
     srcs = [
         "sorting.ts",
-        "vz-sorting.html",
     ],
-    path = "/vz-sorting",
+    strict_checks = False,
 )
 
 tensorboard_webcomponent_library(
```
Comment (stephanwlee) [inline tensorboard/components_polymer3/vz_sorting/BUILD:16]: remove me. I don't think internal rules will be able to handle this dependency.

File: tensorboard/components_polymer3/vz_sorting/test/BUILD [hunk 2]
```diff
@@ -11,6 +11,7 @@ tf_web_test(
     name = "test",
     src = "/vz-sorting/test/tests.html",
     web_library = ":test_web_library",
+    tags = ["manual"]
```
Comment (stephanwlee) [inline tensorboard/components_polymer3/vz_sorting/test/BUILD:14]: buildifier? (trailing comma is missing.)

Comment (psybuzz) [reply tensorboard/components_polymer3/vz_sorting/test/BUILD]: Done, trailing comma and order was off

File: tensorboard/components_polymer3/vz_sorting/BUILD (hunk 1 above)
Comment (psybuzz) [reply tensorboard/components_polymer3/vz_sorting/BUILD]: Done

**Pull request merged**